packages/react-shopify-app-bridge/src/Provider.tsx

```diff
@@ -23,16 +23,11 @@ type GadgetProviderProps = {
   isRootFrameRequest: boolean;
 };
 
-const GetCurrentSessionQuery = `
-  query GetSessionForShopifyApp {
-    currentSession {
-      id
-      shop {
-        id
-      }
-    }
-    shopifyConnection {
-      requiresReauthentication
+const FetchOrInstallShopMutation = `
+  mutation ShopifyFetchOrInstallShop($shopifySessionToken: String!) {
+    shopifyFetchOrInstallShop(shopifySessionToken: $shopifySessionToken) {
```

Comment:
Does it make sense to nest this mutation under the shopifyConnection doodad like previously so as to not pollute their GraphQL namespace?

Reply:
yah definitely